dnf-behave-tests/dnf/api/transaction-simple-install.feature

```diff
@@ -0,0 +1,21 @@
+@dnf5
+@not.with_dnf=4
+Feature: transaction: install a package without dependencies
+
+
+Scenario: Construct query and install labirinto package
+Given I use repository "simple-base"
+ When I execute python libdnf5 api script with setup
+      """
+      goal = libdnf5.base.Goal(base)
+      goal.add_rpm_install("labirinto")
+      execute_transaction(goal, "install a package without dependencies")
+      """
+   Then the exit code is 0
+    And stdout is
+ """
+ labirinto-1.0-1.fc29.x86_64 : Install
+ """
+ When I execute rpm with args "-q labirinto"
```

Comment:
We have a step that checks changes that rpm sees.

We cannot use `Transaction is following` directly because that also parses output of dnf but in both scenarios we can:
```Gherkin
And RPMDB Transaction is following
    | Action        | Package                       |
    | install       | labirinto-0:1.0-1.fc29.x86_64 |
```
The step is checking differences in rpm db.

Reply:
OK, I'll try whether it works. I had sth like this (definitely not exactly "RPMDB Transaction is following") in the code and it didn't work for me.

Reply:
should be fixed now